Makefile

```diff
@@ -84,4 +84,10 @@ install: secvarctl
 	mkdir -p $(DESTDIR)/$(MANDIR)/man1
 	install -m 0644 secvarctl.1 $(DESTDIR)/$(MANDIR)/man1
 
+#dont add all c files, extra mbedtls and skiboot should retain its own format
+format:
+	cp external/linux/.clang-format .
+	clang-format-12 --style=file -i *.c include/*.h crypto/*.c crypto/*.h backends/*/*.c backends/*/*/*.h
```

Comment:
do we need to specify clang-format-12 here? is there a reason 11 or 13 or 9 or 6 won't work?

Reply:
Can I suggest this?

```
 #dont add all c files, extra mbedtls and skiboot should retain its own format
+CLANG_FORMAT ?= clang-format
 format:
        cp external/linux/.clang-format .
-       clang-format-12 --style=file -i *.c include/*.h crypto/*.c crypto/*.h backends/*/*.c backends/*/*/*.h
+       $(CLANG_FORMAT) --style=file -i *.c include/*.h crypto/*.c crypto/*.h backends/*/*.c backends/*/*/*.h
        rm .clang-format
 
 -include $(OBJ:.o=.d)
```

That allows you to override clang-format easily but doesn't require you to have clang-format-12.

Reply:
(I tested with clang-format-11, the results seem the same.)

Reply:
Once this is resolved, I am very strongly a fan of this being merged.

Reply:
Yes. I believe any clang version will work. I had to use clang-format-12 for another `.clang-format` file I was working with (it had some fields and options specific to clang-format-12 apparently). But the current `.clang-format` does not seem to have those newer fields. I just used clang-format-6 and had no issues

Reply:
I will incorporate your recommendation and push the result. Thanks!